index.d.ts

```diff
@@ -541,6 +541,85 @@ console.log(stdout);
 export function execaCommand(command: string, options?: Options): ExecaChildProcess;
 export function execaCommand(command: string, options?: Options<null>): ExecaChildProcess<Buffer>;
 
+type TemplateExpression = string | number | Array<string | number>;
+
+type Execa$<StdoutStderrType = string> = {
+	/**
+  Same as `execa()` except both file and arguments are specified in a single tagged template string. For example, `` $`echo unicorns` `` is the same as `execa('echo', ['unicorns'])`.
+	The `shell` option must be used if the `command` uses shell-specific features (for example, `&&` or `||`), as opposed to being a simple `file` followed by its `arguments`.
+
+	@returns A [`child_process` instance](https://nodejs.org/api/child_process.html#child_process_class_childprocess), which is enhanced to also be a `Promise` for a result `Object` with `stdout` and `stderr` properties.
+
+	@example
+	```
+	import {$} from 'execa';
+	const {stdout} = await $`echo unicorns`;
+	console.log(stdout);
+	//=> 'unicorns'
+
+	const {stdout} = await $`echo ${['unicorns', 'rainbows']}`;
+	console.log(stdout);
+	//=> 'unicorns rainbows'
+
+	await $({stdio: 'inherit'})`echo unicorns`;
+	//=> 'unicorns'
+
+	const $$ = $({stdio: 'inherit'});
+	await $$`echo unicorns`;
+	//=> 'unicorns'
+	await $$`echo rainbows`;
+	//=> 'rainbows'
+	```
+	*/
+	(options: Options<undefined>): Execa$<StdoutStderrType>;
+	(options: Options): Execa$;
+	(options: Options<null>): Execa$<Buffer>;
+	(
+		templates: TemplateStringsArray,
+		...expressions: TemplateExpression[]
+	): ExecaChildProcess<StdoutStderrType>;
+
+	/**
+	Same as `$` but synchronous.
+
+	@returns A result `Object` with `stdout` and `stderr` properties.
```

Comment:
The types already describe this. It could instead say:

```suggestion
	@returns The stdout and stderr output.
```